Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM clojure:tools-deps AS builder
+
+WORKDIR /tmp
+COPY build.clj deps.edn /tmp/
+RUN clj -T:build clean
+# The source code was not include until now intentionally!
+# The above run of "clean" trigerred downloading all dependencies, resulting in a docker layer, that caches our dependencies, as long as they don't change.
+# Without that, each change to any file would trigger a redownload of all dependencies.
+
+COPY Makefile /tmp/
+COPY src /tmp/src
+RUN OAUTH_CLIENT_ID=abc OAUTH_CLIENT_SECRET=xyz make uber
```

Comment:
We could store the `OAUTH_CLIENT_ID` and `OAUTH_CLIENT_SECRET` keys as github Secrets so they can be given to fly as a build secret within the github action from #91 to build the container (see: https://fly.io/docs/reference/build-secrets/). 
And then for running the container we should set these keys as a secret within fly, as described here (https://fly.io/docs/reference/secrets/) is that correct?

Reply:
Thank's for reviewing. The mocked values in the Dockerfile are a workaround until #95 is fixed. They are overriden on runtime.

The actual values are provided to the application as environment variables by fly.io on their servers.
They should not be hard-baked into the application build, since you might want to run multiple instances of that application with different names.

BTW: For the continous deployment (building the latest version of the application after each merge to the main branch and deploying it to fly.io), we use a build secret to authenticate to fly.io. Maybe you could also review the PR #91 for that?